experimental/Origami/src/normal_origami.jl

```diff
@@ -0,0 +1,15 @@
+function normal_stored_origami(h::PermGroupElem, v::PermGroupElem, g::Oscar.GAPGroup)
+    return Normal_stored_origami(GAP.Globals.NormalStoredOrigami(GapObj(h), GapObj(v), g), h, v, g)
+end
+function as_permutation_pepresentation(o::Normal_stored_origami)
+    return GAP.Globals.AsPermutationRepresentation(GapObj(o))
+end
+
+function all_normal_origamis_by_degree(n::Int)
+    return GAP.Globals.AllNormalOrigamisByDegree(n)
```

Comment:
I am guessing this should really return a `Vector{Origami}` and the result should be wrapped a such?

E.g. this might work?
```suggestion
    return [from_GAP_origami(o) for o in GAP.Globals.AllNormalOrigamisByDegree(n)]
```

Reply:
Normal origamis consist of a finite group $G$ and a pair of generators of $G$. See [https://ag-weitze-schmithusen.github.io/Origami/doc/chap2.html#X838AD31B7BF71CDF](https://ag-weitze-schmithusen.github.io/Origami/doc/chap2.html#X838AD31B7BF71CDF). So I'll have to think a little about how to best handle this in the realm of Oscar.

Also, I think the naming of this function is really confusing; I'll talk to @wsinuds about this.